lib/algora/library.ex

```diff
@@ -226,6 +226,29 @@ defmodule Algora.Library do
     |> Repo.one()
   end
 
+  def terminate_stream(video_id) do
+    video = from(v in Video,
+      where: v.id == ^video_id
+    )
+    |> Repo.one()
+
+    {:ok, response} = Finch.build(:get, "#{video.url_root}/g3cFdmlkZW8.m3u8")
```

Comment:
Sometimes this file doesn't exist which causes `get_duration` to crash. Can we add a new field `corrupted` on videos and mark those as such so we can filter them out?